content/arduino-cloud/01.getting-started/02.technical-reference/iot-cloud-tech-ref.md

```diff
@@ -131,6 +131,18 @@ This feature allows Arduino devices to communicate with each other by sharing va
 Using this feature you can link variables of the same data type between **two or more cloud-enabled devices**, and Arduino IoT Cloud will automatically ensure any change to their value is propagated among all linked boards.
 For example, one button could set three smart bulbs to the same color by just setting a variable on one device and reading it on the others. Or you could turn on a heater when temperature sensors in your room or outside in your weather station drop below a certain level.
 
+### Local Time
+
+To retrieve local time, use the `ArduinoCloud.getLocalTime()` function in a sketch. The value returned is the Unix timestamp or UTC, and automatically adjusts based DST (Daylight Time Savings) and the configured time zone. This is changed inside your Thing configuration.
+
+This data can be stored in a `CloudTime` variable, and can be visualed using the [Time Picker widget](/arduino-cloud/getting-started/technical-reference#time-picker).
+
+```arduino
+myTimeVariable = ArduinoCloud.getLocalTime()
+```
+
+***Note that the [Arduino_IoTCloud](https://github.com/arduino-libraries/ArduinoIoTCloud) library creates an instance of `rtc`, using the [RTCZero](https://github.com/arduino-libraries/RTCZero) library. As there is only one hardware instance, you cannot have multiple `rtc` instances.*** 
```

Comment:
```suggestion
***Note that when using a board equipped with a hardware Real-Time Clock (RTC) the [Arduino_IoTCloud](https://github.com/arduino-libraries/ArduinoIoTCloud) library will use it automatically, thus communicating with the RTC from within your sketch or other libaries is not recommended. You can use the `getLocalTime()` and `getInternalTime()` methods provided by Arduino_IoTCloud instead.
*** 
```

Reply:
I support @alranel proposal: RTCZero library is used for MKR boards, but the RTC is used also from mbed boards not using RTCZero.